src/provider/CentralServerProvider.tsx

```diff
@@ -514,15 +511,14 @@ export default class CentralServerProvider {
     return result.data;
   }
 
-  public async getChargingStation(id: string): Promise<ChargingStation> {
+  public async getChargingStation(id: string, extraParams: HttpChargingStationRequest = {}): Promise<ChargingStation> {
     this.debugMethod('getChargingStation');
-    const url = this.buildRestEndpointUrl(ServerRoute.REST_CHARGING_STATION, { id });
+    const params = { ID: id, ...extraParams };
```

Comment:
Je chipote: no need to create an intermediary var here, you can just add one line to the previous existing code (more readable)

```
    const result = await this.axiosInstance.get(url, {
      headers: this.buildSecuredHeaders(),
      params: {
        ID: id
        ...extraParams
      }
```